disnake/channel.py

```diff
@@ -1169,12 +1240,90 @@ def type(self) -> Literal[ChannelType.voice]:
+        as this channel.
+
+        You must have :attr:`.Permissions.manage_channels` permission to
+        do this.
+
+        .. versionchanged:: 2.8
+            Added ``bitrate``, ``user_limit``, ``position``, ``category``,
+            ``rtc_region``, ``video_quality_mode``, ``nsfw``, ``slowmode_delay``, ``flags``
+            keyword-only parameters.
+
+        Parameters
+        ----------
+        name: Optional[:class:`str`]
+            The name of the new channel. If not provided, defaults to this channel name.
+        bitrate: :class:`int`
+            The bitrate of the new channel. If not provided, defaults to this channel bitrate.
+        user_limit: :class:`int`
+            The user limit of the new channel. If not provided, defaults to this channel user limit.
+        position: :class:`int`
+            The position of the new channel. If not provided, defaults to this channel position.
+        category: Optional[:class:`abc.Snowflake`]
+            The category id where the new channel should be grouped. If not provided, defaults to this channel category.
+        rtc_region: Optional[Union[:class:`str`, :class:`VoiceRegion`]]
+            The rtc region of the new channel. If not provided, defaults to this channel rtc region.
+        video_quality_mode: :class:`VideoQualityMode`
+            The video quality mode of the new channel. If not provided, defaults to this channel video quality mode.
+        nsfw: :class:`bool`
+            Wether the new channel should be nsfw or not. If not provided, defaults to this channel nsfw value.
+        slowmode_delay: Optional[:class:`int`]
+            The slowmode of the new channel. If not provided, defaults to this channel slowmode.
+        flags: :class:`ChannelFlags`
+            The flags of the new channel. If not provided, defaults to this channel flags.
+        reason: Optional[:class:`str`]
+            The reason for cloning this channel. Shows up on the audit log.
+
+        Raises
+        ------
+        Forbidden
+            You do not have the proper permissions to create this channel.
+        HTTPException
+            Creating the channel failed.
+
+        Returns
+        -------
+        :class:`VoiceChannel`
+            The channel that was created.
+        """
+
         return await self._clone_impl(
-            {"bitrate": self.bitrate, "user_limit": self.user_limit}, name=name, reason=reason
+            {
+                "bitrate": bitrate if bitrate is not MISSING else self.bitrate,
+                "user_limit": user_limit if user_limit is not MISSING else self.user_limit,
+                "position": position if position is not MISSING else self.position,
+                "rtc_region": rtc_region if rtc_region is not MISSING else self.rtc_region,
+                "video_quality_mode": int(video_quality_mode)
+                if video_quality_mode is not MISSING and video_quality_mode
```

Comment:
```suggestion
                if video_quality_mode is not MISSING
```
Correct me if I'm wrong, but this shouldn't be required